draft-ietf-jsonpath-base.md

```diff
@@ -1485,87 +1494,45 @@ Notes:
 * `LogicalType` is an abstraction of the result of a `logical-expr`.
   Its two instances, `LogicalTrue` and `LogicalFalse`, are not related to
   the JSON literals `true` and `false` and have no syntactical representation in JSONPath.
+* `SingleNodeType` is an abstraction of a `singular-path` (which appears in a comparison,
+  in a test expression, or as a function argument).
```

Comment:
Note to self: it may be necessary to add: 
```suggestion
  in a test expression, or as a function argument).
  Members of `SingleNodeType` have no syntactical representation in JSONPath.
```
but I'm puzzling over the similar statements for `LogicalType` (queried [here](https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/403#discussion_r1122546568)) and `NodesType` (queried [here](https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/403#discussion_r1122548030)).

Reply:
I agree the statement should be there.  My only concern is the possible redundancy of saying it three times.  Maybe there's a way to combine the statements, but I don't mind the repetition.

Reply:
If we decide this PR is the way to go, then we can apply some editorial polish.